build/Helix/UpdateUnreliableTests.ps1

```diff
@@ -54,7 +54,7 @@ foreach ($testRun in $testRuns.value)
             Write-Host "  Test $($testResult.testCaseTitle) was detected as unreliable. Updating..."
             
             # The errorMessage field contains a link to the JSON-encoded rerun result data.
-            $rerunResults = ConvertFrom-Json (New-Object System.Net.WebClient).DownloadString($testResult.errorMessage)
+            $rerunResults = ConvertFrom-Json Download-StringWithRetries $testResult.errorMessage
```

Comment:
This function Download-StringWithRetries takes two arguments. Either update the helper function to only take one argument, or pass a 'file name' here. Also, I am not sure now PS handles precedence, so you might want to add parens. Or do something like:

$resultsJson = Download-StringWithRetries $testResult.errorMessage
$rerunResults = ConvertFrom-Json $resultsJson

Reply:
Good point, addressed both issues now. Going with keeping up the file name to make later WinUI 3 merges easier and help debugging issues down the line.